tests/mk_test.sh

```diff
@@ -156,8 +156,9 @@ function mk_build_Test()
   assertEquals "($ID)" "$ret" "22"
 
   ID=5
-  output=$(mk_build 'TEST_MODE' | head -1) # Remove statistics output
-  expected_result="make -j$PARALLEL_CORES ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu-"
+  output=$(mk_build 'TEST_MODE' | head -2) # Remove statistics output
+  expected_result="make olddefconfig
+make -j$PARALLEL_CORES ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu-"
```

Comment:
I would prefer that you create an array with the command sequence. See `mk_list_remote_kernels_Test` as a reference.